src/components/Footer/styles.js

```diff
@@ -62,7 +70,12 @@ const styles = theme => ({
     color: 'white',
     textTransform: 'lowercase',
     fontFamily: 'Archivo, Arial, sans-serif',
-    fontWeight: 'bold'
+    fontWeight: 'bold',
+
+    [theme.breakpoints.down('sm')]: {
+      padding: 0,
+      minHeight: '0.875rem',
```

Comment:
Also not urgent, but having a a rem that is more granular than a 0.25rem means that were starting to break from a design system into a very specific per component solution. We might need to revisit this, since having once off solutions can get out of hand as codebase grows.

Reply:
I'll make a note of this.